integration/openshift/README.md

```diff
@@ -0,0 +1,27 @@
+# Openshift integration with Clear Containers
+
+In this section you can find scripts to setup an Openshift enviornment
+on top of Clear Containers. Currently, these scripts only work for **Fedora**.
+
+Please execute the steps below on a Clean Environment.
```

Comment:
Use lower case for "clean environment" and remove "Please" to start the sentence with an action.